src/App.vue

```diff
@@ -25,16 +25,6 @@ h1, h2 {
   font-weight: normal;
 }
 
-ul {
-  list-style-type: none;
-  padding: 0;
-}
```

Comment:
This might be ok to keep b/c otherwise the bullets are all the way to the left but the text is centered on the page
<img width="978" alt="screen shot 2018-03-17 at 16 50 46" src="https://user-images.githubusercontent.com/11546231/37559853-74077548-2a03-11e8-8be9-79bfe313089b.png">